ledger/src/proofs/gates.rs

```diff
@@ -298,7 +301,7 @@ pub fn read_constraints_data<F: FieldWitness>(
     // V.t option array list
     let rows_rev = std::fs::read(rows_rev_path).unwrap();
     // std::fs::read(Path::new(env!("CARGO_MANIFEST_DIR")).join("rows_rev.bin")).unwrap();
-    let rows_rev: Vec<Vec<Option<VRaw>>> = Vec::binprot_read(&mut rows_rev.as_slice()).unwrap();
+    let rows_rev: List<List<Option<VRaw>>> = List::binprot_read(&mut rows_rev.as_slice()).unwrap();
```

Comment:
Unsure about these, I changed this code before restoring `Vec` will probably revert it.